src/store/modules/authentication.ts

```diff
@@ -122,14 +119,25 @@ const AuthenticationModule: Module<AuthenticationState, {}> = {
     [AuthenticationActions.STORE_TOKEN] (context: AuthenticationActionContext, token: string) {
       localStorage.setItem(JWT_KEY, token)
     },
-    [AuthenticationActions.LOGIN_WG] (context: AuthenticationActionContext, region: JWTRegion) {
-      // TODO: This is just placeholder logic. Please check it works. -Niko
-      const returnUrl = process.env.VUE_APP_MS_AUTH + `/connect/wargaming/${regionDomain(region)}/${window.location.href}`
+    [AuthenticationActions.LOGIN_WG] (context: AuthenticationActionContext, endpoint: string) {
+      const returnUrl = `${process.env.VUE_APP_MS_AUTH}${endpoint}/${window.location.href}`
       location.assign(returnUrl)
     },
     [AuthenticationActions.LOGOUT] (context: AuthenticationActionContext) {
       localStorage.removeItem(JWT_KEY)
       context.commit(AuthenticationMutation.SET_AUTHENTICATION_TOKEN, null)
+    },
+    async [AuthenticationActions.LOAD_PROVIDERS] (context: AuthenticationActionContext) {
+      if (process.env.VUE_APP_MS_AUTH) {
+        const response = await axios.get(process.env.VUE_APP_MS_AUTH)
+        if (response.status !== 200 || !(response.data?.providers)) {
+          throw Error('Could not fetch authentication providers')
```

Comment:
this needs to be translated

Reply:
This reminds me there's no display in case of error to the user. Should I just put the translation key in the exception message or is there a way to translate from within the vuex store?

Reply:
Leave this for now, I have implemented a way in one of my PRs so I will go back to update this as well once this is merged into dev. I have also closed the other translation comments that are in the store